src/topics/topicSelectors.js

```diff
@@ -27,15 +28,22 @@ export const getTopicsForNarrow: Selector<$ReadOnlyArray<string>, Narrow> = crea
 export const getTopicsForStream: Selector<?$ReadOnlyArray<TopicExtended>, number> = createSelector(
```

Comment:
This selector `getTopicsForStream` gives the data for `TopicListScreen`. Is that the screen meant by this part of #5362's description, under "ideally":

> In muted streams, sorting unmuted topics at the top in the stream topic view.

?

It looks like the web app does that kind of sorting in the left sidebar:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/22248748/234994524-4bfa8234-0532-4d13-9e66-5689d3fe71c9.png">

unless until I click "more topics":

<img width="273" alt="image" src="https://user-images.githubusercontent.com/22248748/234994383-d59ec6fd-417d-47d1-9cb4-3cecee3951bb.png">

Reply:
Hmm, yeah, I'd missed that line in #5362.

It's a bit awkward here, because we don't have the "more topics" distinction — really this screen is always in "more topics" mode.

In particular, in this screen (like in "more topics" in web) we show *all* the topics that have messages, even when already read. So if we just sorted the unmuted topics to the top, then you'd quickly end up having the top clogged with a bunch of old topics that you unmuted back when they were live; effectively it'd degrade to *only* showing you the unmuted topics.

Short of reworking this screen more deeply, I think the right answer here is:
* When you want just the unmuted topics that have unreads (the same list that web puts at the top in the left sidebar), you can find that in the inbox view.
* When you want all the topics (just like "more topics" mode on web), that's this screen.

@alya, thoughts on those UI choices?